Jenkinsfile

```diff
@@ -30,66 +33,100 @@ pipeline {
+
+                                script {
+                                    image_amd64 = docker.build(
+                                        'docker-app-service-configurable',
+                                        "--build-arg MAKE='test build' --label 'git_sha=${env.GIT_COMMIT}' ."
+                                    )
+
+                                    // edgeXCodecov('app-functions-sdk-go-codecov-token')
+                                }
+                            }
+                        }
+                        stage('Docker Push') {
+                            when { expression { edgex.isReleaseStream() } }
+
+                            steps {
+                                script {
+                                    docker.withRegistry("https://${env.DOCKER_REGISTRY}:10004") {
+                                        image_amd64.push(env.VERSION)
+                                        image_amd64.push("${env.GIT_COMMIT}-${env.VERSION}")
+                                    }
+                                }
+                            }
+                        }
                     }
                 }
-                stage('Test arm64') {
+                stage('Build arm64') {
                     agent {
-                        dockerfile {
-                            filename 'Dockerfile.build-arm64'
-                            label 'ubuntu18.04-docker-arm64-4c-2g'
-                            args '-u 0:0 -v /var/run/docker.sock:/var/run/docker.sock' // needed for go mod cache
-                        }
+                        label 'ubuntu18.04-docker-arm64-4c-2g'
                     }
-                    steps {
-                        sh 'make test'
-                        // edgeXCodecov('app-functions-sdk-go-codecov-token')
-                        sh 'make docker'
-                    }
-                }
-            }
-        }
+                    stages {
+                        stage('Docker Build') {
+                            steps {
+                                unstash 'semver'
+                                
+                                sh 'echo Currently Building version: `cat ./VERSION`'
 
-        stage('Semver Init') {
-            when { expression { edgex.isReleaseStream() } }
-            steps {
-                edgeXSemver 'init'
-                script {
-                    def semverVersion = edgeXSemver()
-                    env.setProperty('VERSION', semverVersion)
+                                script {
+                                    image_arm64 = docker.build(
+                                        'docker-app-service-configurable-arm64',
+                                        "--build-arg MAKE='test build' --label 'git_sha=${env.GIT_COMMIT}' ."
+                                    )
+
+                                    // edgeXCodecov('app-functions-sdk-go-codecov-token')
```

Comment:
Remove if no longer needed. Is CodeCov enabled?

Reply:
I need to check with eball to see if they have configured the token